packages/api/src/index.ts

```diff
@@ -17,7 +17,7 @@ import { chalkLog } from '@lib/chalkLog';
 
 const RedisStore = connectRedis(expressSession);
 
-const redisClient = createClient(process.env.OASIS_API_REDIS_URL);
+const redisClient = createClient({url:process.env.OASIS_API_REDIS_URL});
```

Comment:
Please don't change things unrelated to the pr. It just makes it so we can git blame and rollback changes without unintended side effects